### DISCUSSION TITLE
LGTM?

### DISCUSSION THREAD

**Discussion by orcmid** [Interlisp/medley#585] [Q&A] [answered]
I have no idea what that means in approving a pull request.  Apparently lore before my time.

**Comment by lassik** [comment, marked as answer]
"Looks Good To Me", i.e. nothing obviously wrong, can be merged

**Comment by masinter** [comment]
I looked over the diffs. I didn't run the changed system or test it because it wasn't in system critical code or used in the development cycle (as would be for changes that affect the load/makefile/compile cycle (since we're developing on top of previous changes -- one of the risks of the residential style).
